pkgs/by-name/hy/hyp2mat/package.nix

```diff
@@ -25,20 +26,27 @@ stdenv.mkDerivation rec {
     sha256 = "03ibk51swxfl7pfrhcrfiffdi4mnf8kla0g1xj1lsrvrjwapfx03";
   };
 
+  postPatch = lib.optionalString (!stdenv.buildPlatform.canExecute stdenv.hostPlatform) ''
```

Comment:
We should do this unconditional to notice breakages on updates

Reply:
I am not sure it will work then, since the help2man will run before the binary is installed in $out.

Reply:
Actually let me check. It might work.